xbmc/guilib/DirtyRegion.h

```diff
@@ -35,3 +35,14 @@ class CDirtyRegion : public CRect
 };
 
 typedef std::vector<CDirtyRegion> CDirtyRegionList;
+
+class CGUIRenderInfo
+{
+public:
+  CGUIRenderInfo() { Reset(0); }
+  void Reset(unsigned int t) { currentTime = t; dirtyregions.clear(); visible = 0; total = 0; }
+  unsigned int currentTime;
+  CDirtyRegionList dirtyregions;
+  int visible;
+  int total;
```

Comment:
It would be great to have methods to access the member and make them private, like `incrementVisible()`, `addDirtyRegion()` and the necessary getter, but no strong opinion here.